### DISCUSSION TITLE
connect 300 devices in thread mesh network

### DISCUSSION THREAD

**Discussion by suc1** [openthread/openthread#10758] [Q&A] [answered]
The theoretical maximum number of devices in a single Thread mesh network is **16,385**. 
However, some sources suggest that in real-world deployments, the maximum practical number of devices is around **250**.

I need to connect **300** devices. 
Should I aim to set up a single Thread network, or would it be more effective to split them across two subnetworks?

**Comment by jwhui** [comment, marked as answer]
Thread can certinaly support 300 devices in a single network - here's a video demonstrating it: https://www.youtube.com/watch?v=0WXcu_r_lvQ

From an addressing perspective, Thread can support 16,384 devices. However, there are practical constraints such as channel capacity.

All Thread devices for a given network operate on a common IEEE 802.15.4 channel. The PHY data rate is 250 kbps. So it really depends on the overall data rate that the application requires.
